doc/bufr_table_samples/adpupa.nc002102.bufr.table.txt

```diff
@@ -0,0 +1,252 @@
+.------------------------------------------------------------------------------.
+| ------------   USER DEFINITIONS FOR TABLE-A TABLE-B TABLE D   -------------- |
+|------------------------------------------------------------------------------|
+| MNEMONIC | NUMBER | DESCRIPTION                                              |
+|----------|--------|----------------------------------------------------------|
+|          |        |                                                          |
+| NC002102 | A51102 | MSG TYPE 002-102  RAWINSONDE - MOBIL LAND (BUFR)         |
+|          |        |                                                          |
+| WMOBLKST | 301001 | IDENTIFICATION - WMO BLOCK AND STATION NUMBER            |
+| YYMMDD   | 301011 | DATE -- YEAR, MONTH, DAY                                 |
+| HHMMSS   | 301013 |                                                          |
+| LTLONH   | 301021 |                                                          |
+| IDLSIPTM | 301111 | IDENTIFICATION OF LAUNCH SITE AND INSTRUMENTATION        |
+| DATETMLN | 301113 | DATE/TIME OF LAUNCH                                      |
+| HAVCOLS  | 301114 | HORIZONTAL AND VERTICAL COORDINATES OF LAUNCH SITE       |
+| INFORDAS | 301128 | Additional information on radiosonde ascent              |
```

Comment:
Some descriptions are capitalized and some are not. Is this ok?

Reply:
@nicholasesposito same as described above.

Reply:
Got it. I'll approve.